### DISCUSSION TITLE
Numba & llvmlite support

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/meta-tegra#968] [Q&A]
Hey Guys,
Have been struggling with adding python3-numba into the dunfel branch, seems like many dependencies for specific versions of llvm and llvmlite.
I understand most people are running this inside the docker container and not on the target directly but our solution is trying to move out of dockers...
Is there any plan to include these packages in yocto (any branch)?
I kind of built the whole thing successfully but now during runtime i see issues like: +crc is not a recognized feature for this target...

Thanks

**Comment by ichergui** [comment]
Hey @tirankas 
I can work on it when I have time. Is this urgent for you ? 
This recipe should live in [meta-tegra-community](https://github.com/OE4T/meta-tegra-community)
Will keep you updated

**Reply by tirankas** [reply]
Thanks @ichergui !
Nothing is really urgent generally, whatever works for you.
If you need anything let me know, i have been seeing many issues during these integration which i am not sure i selected the right path when fixing these issues.
One thing i noticed about llvm was the way its built requires some exporting of variables to point it to aarch64 rather than letting the recipe pick it up for itself (which it chosses the host arch and bot target arch)

**Reply by ichergui** [reply]
Hello @tirankas 
I'm sorry. I didn't have time to work on introducing `numba` to `meta-tegra-community`
If you have the recipe already. Please push a PR 
I will try to help you solving the issues

**Reply by tirankas** [reply]
Sure @ichergui 
Will do

**Reply by ichergui** [reply]
Perfect

**Comment by proximous** [comment]
Has there been any progress on this? I'm looking to make this work on a different aarch64 system (Xilinx MPSoC) and was hoping to learn from your success here to get it working on my platform. @tirankas can you share what you succeeded with so far?

**Reply by ichergui** [reply]
Hey @proximous 
I don't think there is an updates. Maybe I will take a look on getting something working over the weekend if I have time
I will keep you updated

**Comment by proximous** [comment]
Thanks @ichergui ! Any help would be greatly appreciated! I've been fighting it for several days now and I think solving it might be a bit beyond my skill level.
